ramp-database/rampdb/model/event.py

```diff
@@ -76,110 +159,265 @@ class Event(Model):
+        score : str
+            The combined public score for each fold.
+        """
+        return (None if self.combined_foldwise_valid_score is None
+                else str(round(self.combined_foldwise_valid_score,
+                               self.get_official_score_type(session)
+                                   .precision)))
 
     @property
+    # XXX: This will only work with Flask
     def combined_foldwise_test_score_str(self):
-        return None if self.combined_foldwise_test_score is None else str(
-            round(self.combined_foldwise_test_score,
-                  self.official_score_type.precision))
+        """str: Convert to string the combined public score for each fold."""
+        return (None if self.combined_foldwise_test_score is None
+                else str(round(self.combined_foldwise_test_score,
+                               self.official_score_type.precision)))
+
+    def get_combined_foldwise_test_score_str(self, session):
+        """Convert to string the combined private score for each fold.
+
+        Parameters
+        ----------
+        session : :class:`sqlalchemy.orm.Session`
+            The session used to make the query.
+
+        Returns
+        -------
+        score : str
+            The combined private score for each fold.
+        """
+        return (None if self.combined_foldwise_test_score is None
+                else str(round(self.combined_foldwise_test_score,
+                               self.get_official_score_type(session)
+                                   .precision)))
 
     @property
     def is_open(self):
+        """bool: Whether or not the event is opened."""
         now = datetime.datetime.utcnow()
-        return now > self.opening_timestamp and now < self.closing_timestamp
+        return self.closing_timestamp > now > self.opening_timestamp
 
     @property
     def is_public_open(self):
+        """bool: Whether or not the public phase of the event is opened."""
         now = datetime.datetime.utcnow()
-        return now > self.public_opening_timestamp\
-            and now < self.closing_timestamp
+        return self.closing_timestamp > now > self.public_opening_timestamp
 
     @property
     def is_closed(self):
+        """bool: Whether or not the event is closed."""
         now = datetime.datetime.utcnow()
         return now > self.closing_timestamp
 
     @property
     def n_jobs(self):
-        """Number of jobs for local parallelization.
-
-        return: number of live cv folds.
-        """
+        """int: The number of cv fold which can be used as number of jobs."""
         return sum(1 for cv_fold in self.cv_folds if cv_fold.type == 'live')
 
     @property
     def n_participants(self):
+        """int: The number of participants to the event."""
         return len(self.event_teams)
 
 
 class EventScoreType(Model):
+    """EventScoreType table.
+
+    This is a many-to-many relationship between Event and ScoreType.
```

Comment:
many-to-one?

If I recall the discussion, the ScoreType table was not that useful, and it is actually this table that stores the score types per event? (as one ScoreType record is created per record here) If so, I would mention something about that.